Markdown/Markdown.sublime-syntax

```diff
@@ -203,10 +203,12 @@ contexts:
         - include: table
         - match: ''
           pop: true
-    - match: '^([ ]{0,3})([*+-])(?=\s)'
+    - match: ^([ ]{0,3})([*+-])( (\[[ xX]\]))?(?=\s)
       captures:
         1: markup.list.unnumbered.markdown
         2: markup.list.unnumbered.bullet.markdown punctuation.definition.list_item.markdown
+        3: markup.list.unnumbered.markdown
+        4: keyword.other.checkbox.markdown-gfm
```

Comment:
Random thought while browsing this: What about a `storage` scope?

Reply:
I'd be fine with that.

Reply:
Preference for `storage.type` or `storage.modifier`?

Reply:
Modifier, imo. Actually, now that you mention it it might as well be a language constant. I think I prefer that.

Reply:
> Modifier, imo. Actually, now that you mention it it might as well be a language constant. I think I prefer that.

Done.